egs/swc/s5b/local/chain/compare_wer_general.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
@vesis84, this script is used for preparing tables for comparing the chain systems.. it was copied from ami/s5b.